gulp-tasks/test:build-bundles.js

```diff
@@ -0,0 +1,101 @@
+const path = require('path-extra');
+const gulp = require('gulp');
+const fs = require('fs-extra');
+const rollup = require('rollup-stream');
+const source = require('vinyl-source-stream');
+const resolve = require('rollup-plugin-node-resolve');
+const commonjs = require('rollup-plugin-commonjs');
+const replace = require('rollup-plugin-replace');
+const multiEntry = require('rollup-plugin-multi-entry');
+
+const constants = require('./utils/constants');
+const packageRunnner = require('./utils/package-runner');
+const logHelper = require('./utils/log-helper');
+const pkgPathToName = require('./utils/pkg-path-to-name');
+
+const buildTestBundle = (packagePath, runningEnv, nodeEnv) => {
+  const testPath = path.join(packagePath, 'test');
+  const environmentPath = path.join(testPath, 'bundle', runningEnv);
+
+  // First check if the bundle directory exists, if it doesn't
+  // there is nothing to build (NOTE: Rollup + multientry will
+  // always create a file, even if the directory doesn't exist)
+  if (!fs.pathExistsSync(environmentPath)) {
+    return Promise.resolve();
+  }
+
+  logHelper.log(`Building Test Bundle for ` +
```

Comment:
The major fear I have is that if we add this into out gulp tasks, it's a short step for someone to want / ask / use it in the actual library, in which case we are taking on extra code for no real reason.